src/condor_shadow.V6.1/pseudo_ops.cpp

```diff
@@ -1072,11 +1104,16 @@ start_input_transfer_failure_conversation( ClassAd request ) {
 
 
 ClassAd
-UniShadow::do_common_file_transfer(
-	const ClassAd & /* request */,
-	const std::string & commonInputFiles,
-	const std::string & cifName
+UniShadow::before_common_file_transfer(
+	const std::string & cifName,
+	const std::string & commonInputFiles
 ) {
+	dprintf( D_ALWAYS, "Starting common files transfer.\n" );
```

Comment:
Should we dprintf the name here?

Reply:
I don't honestly know.  The names at this point are already the "internal" names, which won't be generally useful.  Arguably, the "external" names should be plumbed through to the job log event(s), but for the starter log, I'm genuinely unsure.  I could plumb the "external" names through to here, I guess.  (Should the "internal" names be reserved for a higher debug level?)

Reply:
I can live without dprintf'ing the names.